modules/virt-about-resource-quota-limits.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-specifying-nodes-for-vms.adoc
+
+:_content-type: CONCEPT
+[id="virt-about-resource-quota-limits_{context}"]
+= About resource limits
```

Comment:
However it is worded, but the title should refer to working with ResourceQuotas

Reply:
I changed the ID and title to "Setting resource quota limits" and made this a PROCEDURE module called by the "Working with resource quotas" assembly.

@ousleyp - Do you think I should retain "in {VirtProductName}" in the assembly and module titles to differentiate from OCP or do you think that's just redundant?

Reply:
I don't think we need "in {VirtProductName}" but I wouldn't be opposed to you adding something like "...for virtual machines" to a title. I'm kind of ambivalent, though, since it's in the CNV docs

Reply:
I'll probably include "for VMs" just for findability.